Docs/developer_guide/build_instructions/windows.md

```diff
@@ -8,7 +8,10 @@ Slicer relies on a number of large third-party libraries (such VTK, ITK, DCMTK),
 
 ## Install prerequisites
 
-- [CMake](https://www.cmake.org/cmake/resources/software.html) version that meets at least the minimum required CMake 3.21.1 <= version [< 3.25.0](https://gitlab.kitware.com/cmake/cmake/-/issues/24180) (or 3.16.3 <= version < 3.20.4)
+- [CMake](https://www.cmake.org/cmake/resources/software.html) version >= 3.16.3.
+  - Versions with known Slicer build issues:
+    - 3.20.4 to 3.21.0
```

Comment:
I suggest we update the at the top of the `CMakeLists.txt` and also reference the corresponding issue/discussion.

Reply:
Will push this change later today